src/chats/components/conversation/footer/EmojiPicker.tsx

```diff
@@ -98,6 +100,11 @@ const EmojiPicker: React.FC<EmojiPickerProps> = ({
 		};
 	}, [setShowEmojiPicker, emojiTimeoutRef, mouseEnterEvent, mouseLeaveEvent]);
 
+	const scaleHeight = useMemo(
+		() => (isInsideMeeting ? 290 / calcScaleDivisor() : 435 / calcScaleDivisor()),
```

Comment:
Here it will be better to store this value of pixel that represent the height of emoji picker inside a variable. Doing so we will understand better what actually is this number and also we can manage all these values inside a unique global variables file.